subfun/setSpeedTargets.m

```diff
@@ -0,0 +1,23 @@
+% (C) Copyright 2021 CPP visual motion localizer developpers
+
+% [ W I P ]
+
+function speeds = setSpeedTargets(cfg)
+
+    % Compute the matrix with the speed targets if requested, otherwise output will be only zeros
+    if sum(contains(cfg.target.type, 'speed')) ~= 0
```

Comment:
won't touch this for now because at the moment either "side" of the if block does the same.